manifests/mod/auth_cas.pp

```diff
@@ -0,0 +1,48 @@
+class apache::mod::auth_cas (
+  $cas_login_url            = undef,
+  $cas_validate_url         = undef,
```

Comment:
Since these two variables are required, they should not be set to undef but rather have no default value.